pilot/pkg/model/push_context.go

```diff
@@ -145,11 +148,17 @@ type UpdateRequest struct {
 	// Full determines whether a full push is required or not. If set to false, only endpoints will be sent.
 	Full bool
 
+	// ConfigTypesUpdated contains the types of configs that have changed.
+	// The config types are those defined in config.go
+	// Applicable only when Full is set to true.
+	ConfigTypesUpdated map[string]struct{}
```

Comment:
if this is empty then what is behavior? Assume any/all configs updated?

Reply:
yes.. if empty, all are updated